src/Schema/ScreenOwnerType.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * All valid screen (page) types in Eigen
+ * @packageDocumentation
+ */
+export enum ScreenOwnerType {
+  Artist = "Artist",
+  Artwork = "Artwork",
+  Auctions = "Auctions",
+  Consign = "Consign",
+  CityGuideGuide = "CityGuideGuide",
+  CityGuideMap = "CityGuideMap",
+  CityPicker = "CityPicker",
+  Fair = "Fair",
+  Inbox = "Inbox",
+  Sale = "Sale",
+  Search = "Search",
+  SavesAndFollows = "SavesAndFollows",
+  ViewingRoom = "ViewingRoom",
+  ViewingRoomArtworks = "ViewingRoomArtworks",
```

Comment:
@louislecluse i've added these screen types here re: viewing rooms, in case you need to edit or reference